### PR TITLE
GH-121: Kafka Message Driven Channel Adapter Typo

### PR DESCRIPTION
**Pull request by magiccrafter**

### CODE REVIEW

File: src/main/java/org/springframework/integration/dsl/kafka/Kafka.java [hunk 1]
```diff
@@ -81,23 +82,52 @@ public static KafkaProducerMessageHandlerSpec outboundChannelAdapter(Properties
...
+	@Deprecated
 	public static KafkaMessageDrivenChannelAdapterSpec messageDriverChannelAdapter(
 			KafkaMessageListenerContainer messageListenerContainer) {
 		return new KafkaMessageDrivenChannelAdapterSpec(messageListenerContainer);
 	}
 
 	/**
+	 * Create an initial {@link KafkaMessageDrivenChannelAdapterSpec}.
+	 * @param messageListenerContainer the {@link KafkaMessageListenerContainer}.
+	 * @return the KafkaMessageDrivenChannelAdapterSpec.
```
Comment (artembilan) [inline src/main/java/org/springframework/integration/dsl/kafka/Kafka.java]: Since this is a new API we should provide a `@since 1.2`

**Comment by magiccrafter** [comment]
Thanks for the review, Artem.
Totally agree with all comments. Changes were applied respectively.

**Comment by artembilan** [comment]
Yeah! I see your changes.

Thanks!

Will review soon and merge if that.

**Comment by artembilan** [comment]
Merged as https://github.com/spring-projects/spring-integration-java-dsl/commit/330ca2b86559aa1ec4d6d6802922e548ec706838. 

Thank you for the contribution.
Looking forward for more!

**Pull request closed without merging**